src/components/PageElements/PageElements.tsx

```diff
@@ -100,90 +118,244 @@ const PageElements: React.FC<PageElementProps> = ({
-              </div>
-            )
-          }
-        )}
-      </Form>
+                  ) : null,
+              }
+
+              const toggleDecision = openResponse === element.code
+
+              return (
+                <div key={`${element.code}ReviewContainer`}>
+                  <Segment basic key={`question_${element.id}`} className="summary-page-element">
+                    {element.category === TemplateElementCategory.Information ? (
+                      <SummaryInformationElement {...summaryViewProps} />
+                    ) : (
+                      // TODO: Show consolidator comment to Reviewer with button to Update
+                      <ReviewApplicantResponse {...reviewProps} {...summaryViewProps} />
+                    )}
+                  </Segment>
+                  {toggleDecision && <DecisionArea {...reviewProps} {...summaryViewProps} />}
+                </div>
+              )
+            }
+          )}
+        </Form>
+      </div>
     )
   }
+
   return null
 }
 
+type ConsolidateReviewDecisionProps = ReviewElementProps & SummaryViewWrapperProps
+const ConsolidateReviewDecision: React.FC<ConsolidateReviewDecisionProps> = ({
+  reviewResponse,
+  isConsolidation,
+  originalReviewResponse,
+  ExtraReviewAction,
+  ExtraEditReviewAction,
+  ...otherProps
+}) => (
+  <div>
+    <ApplicantResponseElement {...otherProps} />
+    <ReviewDecisionElement
+      isActiveReview={true}
+      reviewResponse={reviewResponse}
+      originalReviewResponse={originalReviewResponse}
+      isConsolidation={isConsolidation}
+      {...otherProps}
+    >
+      {ExtraEditReviewAction && <ExtraEditReviewAction />}
+    </ReviewDecisionElement>
+    <ReviewDecisionElement
+      isActiveReview={false}
+      reviewResponse={originalReviewResponse as ReviewResponse}
+      isConsolidation={false}
+      {...otherProps}
+    >
+      {ExtraReviewAction && <ExtraReviewAction />}
+    </ReviewDecisionElement>
+  </div>
+)
+
+type ReviewApplicantResponseProps = ReviewElementProps & SummaryViewWrapperProps
+
+const ReviewApplicantResponse: React.FC<ReviewApplicantResponseProps> = ({
+  reviewResponse,
+  ExtraReviewAction,
+  ExtraEditReviewAction,
+  ...otherProps
+}) => (
+  <div>
+    <ApplicantResponseElement {...otherProps}>
+      {ExtraReviewAction && <ExtraReviewAction />}
+    </ApplicantResponseElement>
+    <ReviewDecisionElement isActiveReview={true} reviewResponse={reviewResponse} {...otherProps}>
+      {ExtraEditReviewAction && <ExtraEditReviewAction />}
+    </ReviewDecisionElement>
+  </div>
+)
+
+const UpdateIcon: React.FC<{ onClick: Function }> = ({ onClick }) => (
+  <Icon className="clickable" name="pencil" size="large" color="blue" onClick={onClick} />
+)
+
+interface ReviewButtonProps {
+  isActive: boolean
+  isNewReview: boolean
+  elementCode: string
+  updateQuery: Function
+}
+
+const ReviewButton: React.FC<ReviewButtonProps> = ({
+  isActive,
+  isNewReview,
+  elementCode,
+  updateQuery,
```

Comment:
Not sure. Have to think about - this PR involved too many decisions (and finding out why bugs were happening) so I tried to avoid adding more to it. But once all other changes are made we can see if this can be addressed as part of it.

Reply:
I'm copying this comment to the new issue to move the decision to inline on the Review page : )